<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_main</class>
 <widget class="QMainWindow" name="frm_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>643</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>KaiQt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tv_queue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>781</width>
      <height>161</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Queue</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_edit_url">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>dank music links or playlist links</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_analyse">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>250</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyse and add to queue</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_format">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>.mp3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.flac</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>URL </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>230</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Default format"/>
    </property>
    <property name="text">
     <string>Format</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tv_results">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>781</width>
      <height>161</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Results</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_dl_convert">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>371</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Download and convert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_cancel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>550</y>
      <width>381</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>781</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_analyse_log">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>219</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_progress">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>515</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_quality">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>320 kbps</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>192 kbps</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>128 kbps</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>96 kbps</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32 kbps</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16 kbps</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Default format"/>
    </property>
    <property name="text">
     <string>Quality</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_auto_download">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>290</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto-Download</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPick_download_folder">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="action_pick_download_location"/>
   </widget>
   <addaction name="menuPick_download_folder"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_pick_download_location">
   <property name="text">
    <string>Pick download location</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
